SCClassLibrary/Platform/osx/OSXPlatform.sc

```diff
@@ -76,4 +76,12 @@ OSXPlatform : UnixPlatform {
 		file.putString(string);
 		file.close;
 	}
+
+	defaultTempDir {
+		var dir;
+		// ensure trailing slash
+		dir = Platform.userAppSupportDir +/+ "tmp/";
+		if (File.exists(dir).not, { dir.mkdir; });
```

Comment:
please fix formatting - `if(File.exists(dir).not) { dir.mkdir };`

Reply:
Wow, I did not know this syntax was allowed !
```
if (bool) { "true".postln } { "false".postln }
```

I always thought parenthesis around functions were needed, because if was a function that would take a boolean and two functions.

```
if (bool, { "true".postln }, { "false".postln });
```

This syntax is not mentionned in http://doc.sccode.org/Reference/Control-Structures.html

Should I fill a separate PR for that ?

Reply:
also, why no space between if and the condition ?
is there a guide somewhere for the codebase formatting ?

Reply:
There's this WIP guide. I seems to contradict what is being suggested here: https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#whitespace-1

Reply:
After the discussion on Slack, the style guide mentions a space after if for the C++ codebase, but no space for the SCLang codebase.

Then I will use 

```supercollider
if(File.exists(dir).not) { dir.mkdir };
```

Reply:
changes done